src/translations/pl.json

```diff
@@ -0,0 +1,27 @@
+{
+  "activityDetails": "Nauczyciel: {teacher}, w sali: {room}",
+  "activityFormBtnEdit": "Edytuj Aktywność",
```

Comment:
i don't like the way you organized it.
At first, you can make steps, like:
```
activities: {
    editActivity: "...."
weekDays: {
...
},
 shared: {
.....
 }
}
```
Second think. InputDesc-Room are not good. You shouldn't name keys like `inputPlaceholder`, this translation can be used not only in placeholder. 
It is correct name ? `"inputDesc-Activity Name"`, I mean space there? try to name it like keys in objects